test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -236,12 +236,33 @@ public void create() {
           Arrays.stream(nodes).mapToObj(n -> "" + n).collect(Collectors.joining(":")),
```

Comment:
All changes in this file aim to address configuration changes in Cassandra 4.1 brought in by CASSANDRA-15234